composer.json

```diff
@@ -16,33 +16,30 @@
         }
     ],
     "require": {
-        "php": "~5.5|~7.0",
-        "sonata-project/admin-bundle": "~3.0",
-        "sonata-project/block-bundle": "^2.3.2",
-        "symfony/framework-bundle": "~2.3|~3.0",
-        "symfony/property-access": "~2.2|~3.0",
-        "doctrine/phpcr-odm": "~1.1",
-        "doctrine/phpcr-bundle": "~1.1",
-        "symfony-cmf/tree-browser-bundle": "~2.0",
-        "symfony-cmf/resource-rest-bundle": "~1.0"
+        "php": "^5.5 || ^7.0",
+        "sonata-project/admin-bundle": "3.x-dev@dev",
+        "symfony/framework-bundle": "^2.3 || ^3.0",
+        "symfony/property-access": "^2.2 || ^3.0",
+        "doctrine/phpcr-odm": "^1.1",
+        "doctrine/phpcr-bundle": "^1.1",
+        "symfony-cmf/tree-browser-bundle": "^2.0@dev",
```

Comment:
that makes no difference, it still opens the dependency to dev versions. 

in the cmf, we preferred to use the minimum-stability during development and remove that before releaseing (when the dependencies hopefully have stable versions around). actually keeping minimum-stability can reveil accidental BC breaks in the dependencies. as it only affects composer run in this repository and not the including project, i find it cleaner than adding `@dev` into dependencies. but if somebody already starts using this, they need to include all our deps explicitly or also set minimum-stability in the project to end up with something installable. so really not sure which is the more elegant approach.